packages/eslint-config-algolia/rules/react.js

```diff
@@ -10,7 +10,112 @@ module.exports = {
+      {
+        declaration: 'parens',
+        assignment: 'parens',
+        return: 'parens',
+        arrow: 'parens',
+        condition: 'ignore',
+        logical: 'ignore',
+        prop: 'ignore',
+      },
+    ],
+    'react/jsx-props-no-spreading': ['off'],
+    'react/jsx-fragments': ['error'],
+    'react/jsx-handler-names': ['error'],
+    'react/no-access-state-in-setstate': ['error'],
+    'react/no-unescaped-entities': ['error'],
+    'react/no-unused-state': ['error'],
+    'react/prop-types': ['off'],
+    'react/sort-comp': [
+      'off',
+      {
+        order: ['lifecycle', 'static-methods', 'everything-else', 'render'],
+      },
+    ],
+    'react/static-property-placement': ['error'],
+    'react/function-component-definition': [
+      'error',
+      {
+        namedComponents: 'arrow-function',
+      },
+    ],
+    'react/no-array-index-key': ['warn'],
+    'react/no-deprecated': ['error'],
+    'react/no-did-mount-set-state': ['warn'],
+    'react/no-did-update-set-state': ['warn'],
+    'react/no-direct-mutation-state': ['error'],
+    'react/no-find-dom-node': ['error'],
+    'react/no-is-mounted': ['error'],
+    'react/no-multi-comp': ['warn', { ignoreStateless: true }],
+    'react/no-redundant-should-component-update': ['error'],
+    'react/no-string-refs': ['error'],
+    'react/no-this-in-sfc': ['error'],
+    'react/no-typos': ['error'],
+    'react/prefer-stateless-function': ['warn'],
+    'react/self-closing-comp': ['error', { html: false }],
+    'react/jsx-sort-props': [
+      'error',
+      {
+        callbacksLast: true,
+        shorthandFirst: true,
+        shorthandLast: false,
+        ignoreCase: true,
+        noSortAlphabetically: true,
+        reservedFirst: false,
+      },
+    ],
+    'react/jsx-uses-react': 'error',
+    'react/jsx-uses-vars': 'error',
+    'react/react-in-jsx-scope': 'error',
+
+    // ---- JSX-a11y
+    'jsx-a11y/alt-text': ['error'],
+    'jsx-a11y/anchor-is-valid': ['error'],
+    'jsx-a11y/anchor-has-content': ['error', { components: [] }],
+    'jsx-a11y/aria-activedescendant-has-tabindex': ['error'],
+    'jsx-a11y/aria-props': ['error'],
+    'jsx-a11y/aria-proptypes': ['error'],
+    'jsx-a11y/aria-role': ['error', { ignoreNonDom: false }],
+    'jsx-a11y/click-events-have-key-events': ['off'],
+    'jsx-a11y/control-has-associated-label': ['off'],
+    'jsx-a11y/heading-has-content': ['off'],
+    'jsx-a11y/html-has-lang': ['error'],
+    'jsx-a11y/iframe-has-title': ['error'],
+    'jsx-a11y/interactive-supports-focus': ['error'],
+    'jsx-a11y/label-has-associated-control': ['off'],
+    'jsx-a11y/lang': ['error'],
+    'jsx-a11y/media-has-caption': ['error'],
+    'jsx-a11y/mouse-events-have-key-events': ['error'],
+    'jsx-a11y/no-access-key': ['error'],
+    'jsx-a11y/no-autofocus': ['error', { ignoreNonDOM: true }],
+    'jsx-a11y/no-distracting-elements': [
+      'error',
+      {
+        elements: ['marquee', 'blink'],
+      },
+    ],
```

Comment:
do we really need this rule? blink doesn't even exist anymore

Reply:
it's the recommended settings